src/Upgrade/Application/Strategy/ReleaseApp/ReleaseGroupFilter/ReleaseGroupFilterInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\ReleaseApp\ReleaseGroupFilter;
```

Comment:
Why can't we use one Interface for both strategies? pls move it to the abstract layer, and use the one